src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1534,6 +1534,8 @@ namespace UnitarySystems {
         } else {
             state.dataHVACGlobal->DXCT = 1; // uses normal DX coil flow limits
         }
+        // sizing may need to know what type of coil is being sized
+        state.dataSize->DataCoolCoilType = this->m_CoolingCoilType_Num;
```

Comment:
UnitarySystem sets this to mimic the sizing of the VS coil model. Currently only cooling is accounted for.